docs/releases/5.2.txt

```diff
@@ -167,6 +167,10 @@ Forms
 
 * The new :class:`~django.forms.SearchInput` form widget is for entering search
   queries and renders as ``<input type="search" ...>``.
+* Added :class:`~django.forms.TelInput` form widget that renders as ``<input type="tel" ...>``
```

Comment:
I think this is redundant now that present tense snippet below it has been added
```suggestion
```